File: dayOne/JavaScript/LevelOne/NonPerfectSquareNumbers.js
function replaceNonPerfectSquares(numbers) {
    const result = [];
        
    for (const number of numbers) {
        const sqrt = Math.floor(Math.sqrt(number));
            
        if (sqrt * sqrt === number) {
            result.push(number);
        } else {
            result.push(-1)
        }
    }
            
    
    return result;
}

const array = [4, 7, 9, 10, 49, 6];
const newArray = replaceNonPerfectSquares(array);

console.log(newArray);

File: dayOne/Python/LevelOne/non_perfect_square_numbers.py
import math

def replace_non_perfect_square_numbers(list):

    result = []
    
    for number in list:
        sqrt = int(math.sqrt(number))
            
        if sqrt * sqrt == number:
            result.append(number)
        else:
            result.append(-1)
                
    return result
    
numbers = [4, 7, 9, 10, 49, 6]
result = replace_non_perfect_square_numbers(numbers)

print("New List: ", result)
